src/components/NcListItemIcon/NcListItemIcon.vue

```diff
@@ -127,15 +127,14 @@ It might be used for list rendering or within the multiselect for example
 <template>
 	<span :id="id"
 		class="option"
-		:style="cssVars"
-		v-on="$listeners">
-		<NcAvatar v-bind="$attrs"
+		:style="cssVars">
+		<!-- <NcAvatar v-bind="$attrs"
 			:disable-menu="true"
 			:disable-tooltip="true"
 			:display-name="displayName || name"
 			:is-no-user="isNoUser"
 			:size="avatarSize"
-			class="option__avatar" />
+			class="option__avatar" /> -->
```

Comment:
`NcAvatar` relies on `NcActions`, which relies on `NcSelect`. So I commented it out here, to not have a giant PR with actions, datepicker and select together. I will bring it back once `NcActions` are migrated.